scripts/generate_special_function_declarations.py

```diff
@@ -84,14 +84,14 @@ def parse_capsule_name(capsule):
     # There isn't a Python equivalent to `PyCapsule_GetName`, so
     # resort to a hacky method for finding the signature.
     match = re.match(
-        '\<capsule object "(?P<signature>.+)" at 0x[a-f0-9]+\>',
+        '<capsule object "(?P<signature>.+)" at 0x[a-f0-9]+>',
```

Comment:
I am uneasy about this change. The one below seems to have broken things.

Reply:
`<` is not a special character in regex so this should be harmless but needless. At best this would have to go inside a raw string literal (because `'\<'` is not a valid string escape), but I'm pretty sure we can revert this change.

Reply:
(Deleted a previous comment because I misread the order of the change here.)

`<` is not a special character in regex so the original was harmless but needless. At best this should have to gone inside a raw string literal (because '\<' is not a valid string escape), but this correction is valid I think.